### DISCUSSION TITLE
help to add RTL support for admin UI

### DISCUSSION THREAD

**Discussion by DasMergo** [directus/directus#18355] [Feature Requests]
### Summary

Related to the following request #17602 , we still need support for non left to right languages such as Kurdish (Sorani), Georgian, Hebrew, Persian , Arabic and other right to left languages, I would like to help you @halwesit on adding RTL support. so let me know if you interested.

### Motivation

this will be the first RTL supported Headless CMS in market that bring many users.

### Must Have:
- an option for dynamic direction change in admin panel (when user switched language) the direction changes at the same time something that injects dir attribute to <html> element like `<html dir="rtl">` and also `<html dir="ltr">` for English and non RTL languages

### Drawbacks

The experience of many right-to-left language users will be bad.

**Comment by halwesit** [comment]
Thanks for your support, of course, Directus deserves to be the first Headless CMS to implement RTL support for the Admin UI.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
